pyproject.toml

```diff
@@ -104,13 +105,20 @@ version-file = "anndata/_version.py"
 
 [tool.coverage.run]
 source = ["anndata"]
-omit = ["setup.py", "versioneer.py", "anndata/_version.py", "**/test_*.py"]
+omit = ["anndata/_version.py", "**/test_*.py"]
 
 [tool.pytest.ini_options]
-addopts = "--doctest-modules"
+addopts = [
+    "--strict-markers",
+    "--doctest-modules",
+    "--ignore=anndata/core.py",      # deprecated
+    "--ignore=anndata/readwrite.py", # deprecated
+]
+filterwarnings = [
+    "ignore::anndata._warnings.ExperimentalFeatureWarning",
```

Comment:
Is this meant to be in both the job definition and here?

Reply:
yes, as discussed, if we want to ignore a warning globally, we should ignore it globally in all jobs.